### ISSUE TITLE
Tracker Results

### ISSUE THREAD

**Issue by fuankarion** [zcxu-eric/Ego4d_TalkNet_ASD#5]
Hi,

I was wondering if the tracking results are available somewhere to run inference on EGO4D.

Thanks!

**Comment by zcxu-eric** [comment]
Hi, there is no available tracking results because this is the ground truth of one Ego4D challenge.

**Comment by fuankarion** [comment]
Thanks for the quick response!, so if I want to reproduce the 79.27% accuracy, should I use the bounding boxes provided in the json file?

**Comment by zcxu-eric** [comment]
Yes, and we provide the cropped images and audio streams for inference.

**Comment by fuankarion** [comment]
Great, just a final question, do you mean you have a download link for the crops and audio? or is it by the slicing scripts on the code?

**Comment by zcxu-eric** [comment]
Please check the instructions: https://github.com/EGO4D/social-interactions/tree/lam#4-test and https://github.com/EGO4D/social-interactions/tree/ttm#4-test.
